RELEASE.rst

```diff
@@ -0,0 +1,19 @@
+RELEASE_TYPE: patch
+
+This release makes several changes:
+
+1. It significantly improves Hypothesis's ability to use coverage information
+   to find interesting examples.
+2. It reduces the default ``max_examples`` setting to 100. This is intended to
```

Comment:
On the basis that most readers know very little about Hypothesis internals, I'd add some context and take out the caveat:

> It reduces the default ``max_examples`` setting to from 200 to 100. This offsets ...